articles/scp-3211/src/en/anomalies/cat.ts

```diff
@@ -0,0 +1,35 @@
+import { Anomaly } from "../../anomaly"
+
+export default new Anomaly(false, {
+  imageUrl: "cat.jpg",
+  imageCaption: "SCP-3211 photographed in containment.",
+  objectClass: "Safe",
+  physicalDescription: `
+    SCP-3211 is a male Bengal cat that is estimated to be about
+    3 years old. It appears with a color and metal tag around its neck.
```

Comment:
"color and metal"? Possibly 'collar'?

What does "it appears" mean here?

Reply:
Wow, I'm tired and misspelled "collar". sad